Composer/ScriptHandler.php

```diff
@@ -394,6 +400,8 @@ protected static function getOptions(Event $event)
 
         $options['symfony-assets-install'] = getenv('SYMFONY_ASSETS_INSTALL') ?: $options['symfony-assets-install'];
 
+        $options['symfony-env'] = getenv('SYMFONY_ENV') ?: $options['symfony-env'];
```

Comment:
can you pass this just before `symfony-assets-install` without blank line between them?